documentation/modules/proc-accessing-kafka-using-nodeports.adoc

```diff
@@ -35,36 +35,38 @@ spec:
         authentication:
           type: tls
         configuration:
-          brokerCertChainAndKey: <1>
+          brokerCertChainAndKey: <2>
             secretName: my-secret
             certificate: my-certificate.crt
             key: my-key.key
```

Comment:
This is unrelated to the nodeports. MAybe we should remove it from this procedure?

Reply:
We can remove it, but isn't it used with nodeport? https://strimzi.io/docs/operators/master/using.html#type-NodePortListenerConfiguration-reference

Reply:
It can be used with any external listener. So it is not wrong per se. But it is not required, just optional. So I think having it in this example just here might be confusing.

Reply:
Removed. We talk about it in the concepts and _Providing your own Kafka listener certificates_. I guess that's enough.